civihr_employee_portal/civihr_leave_absences/civihr_leave_absences.module

```diff
@@ -60,3 +68,23 @@ function civihr_leave_absences_my_leave_get_markup() {
 
   return '';
 }
+
+/**
+ * Internal function for fetching custom markup from template/Angular Js to display
+ * Manager Leave block.
+ * @return String
+ *    Custom markup from template file.
+ */
+function civihr_leave_absences_manager_leave_get_markup() {
+
+  // @ TODO -> Pointing to Angular JS App once its ready for Manager Leave block.
+  // Loading angular js application for Manager Leave block using CiviCRM Resources API.
+  // eg: civicrm_resources_load('org.civicrm.hrjobcontract', array('gulpfile.js', 'hrjc.css', 'contact.js'));
+
+  // Fetching custom markup from pre-defined template file.
+  $file_path = drupal_get_path('module', 'civihr_leave_absences') . '/templates/manager_leave.html';
```

Comment:
You are starting to have duplicated code. Please move this
```php
$file_path = drupal_get_path('module', 'civihr_leave_absences') . "/templates/#{tplName}.html";

if (file_exists($file_path)) {
   return @file_get_contents($file_path);
 } else {
  return '';
}
```
in a private function so that both `civihr_leave_absences_manager_leave_get_markup` and `civihr_leave_absences_my_leave_get_markup` can call it

(i'm pretty sure that the whole thing could be abstracted even more, but let's start with this at least)